backend/internal/model/user.go

```diff
@@ -13,14 +13,15 @@ import (
 type User struct {
 	Base
 
-	Username  string `sortable:"true"`
-	Email     string `sortable:"true"`
-	FirstName string `sortable:"true"`
-	LastName  string `sortable:"true"`
-	IsAdmin   bool   `sortable:"true"`
-	Locale    *string
-	LdapID    *string
-	Disabled  bool `sortable:"true"`
+	Username    string  `sortable:"true"`
+	Email       string  `sortable:"true"`
+	FirstName   string  `sortable:"true"`
+	LastName    *string `sortable:"true"`
```

Comment:
Why changing this to a pointer? I don't think that's needed, and makes things more complex... A null last name or a zero value one should be treated the same
```suggestion
	LastName    string `sortable:"true"`
```

Reply:
`LastName` can be null, and Gorm suggests using pointers for [nullable fields](https://gorm.io/docs/models.html#Declaring-Models). I'm not a fan of this but there isn't really another way.